src/excmds.ts

```diff
@@ -2946,38 +2949,7 @@ export async function shellescape(...quoteme: string[]) {
  */
```

Comment:
These docs need updating - how to rebind and also the different behaviour between calling from an RC file vs calling via a browser-mode bind.